components/mobile-stepper-example.tsx

```diff
@@ -34,7 +36,7 @@ export const MobileStepperExample: React.FC = () => {
 
     return (
         <Card style={{ padding: 0, margin: PADDING, marginBottom: PADDING }}>
-            <Button mode="contained" onPress={(): void => cycleStepperVariant()}>
+            <Button mode="contained" onPress={(): void => cycleStepperVariant()} theme={theme.dark ? blueDarkAlt : {}}>
```

Comment:
We should be using the Wrapper versions of components _everywhere_ in place of the RNP components.